server/src/model/external_events.go

```diff
@@ -33,15 +33,63 @@ type ExternalEvent struct {
 	Featured        bool      `db:"featured"`
 }
 
-func GetExternalEvents(db *sqlx.DB, pageID int, startTime time.Time, endTime time.Time, onlineOnly bool) ([]ExternalEvent, error) {
+func GetExternalEventsWithFallback(db *sqlx.DB, pageID int, startTime time.Time, endTime time.Time) (events []ExternalEvent, localEventsFound bool, err error) {
+	localEventsFound = false
+
+	// run query to get local events
+	if IsBayAreaPage(pageID) {
+		// If one Bay Area page is chosen, combine events from all Bay Area pages
+		for _, chapterPageID := range BayAreaPages {
+			chapterEvents, err := GetExternalEvents(db, chapterPageID, startTime, endTime)
```

Comment:
if an event is co-hosted by both chapters, will it show up twice?

Reply:
fixed!